.github/no-response.yml

```diff
@@ -0,0 +1,10 @@
+#Configuration for probot-no-response - https://github.com/probot/no-response
+
+#Number of days of inactivity before an Issue is closed for lack of response
+daysUntilClose: 21
```

Comment:
@shreyasbapat Very auspicious number it is !

Reply:
I know right?